lib/chef-acceptance/chef_runner.rb

```diff
@@ -74,19 +74,17 @@ def chef_config_template
           #{File.expand_path('..', acceptance_cookbook.root_dir).inspect},
           #{File.expand_path('../../../.shared', acceptance_cookbook.root_dir).inspect}
         ]
-        node_path #{File.expand_path('nodes', acceptance_cookbook.root_dir).inspect}
+        node_path "#{node_path}"
         stream_execute_output true
         audit_mode #{app_options.audit_mode ? ":enabled" : ":disabled"}
```

Comment:
In a traditional chef-client run there will only be a single process writing to and reading from `/var/chef/cache`. In this situation multiple parallel processes will be reading from and writing to this shared state/cache. This could produce some hard to debug errors which can be avoided by just allowing each Chef Acceptance suite's chef-client run to have its own dedicated cache.